classifier.go

```diff
@@ -10,7 +10,7 @@ import (
 // The scores must order the candidates so as the highest score be the most probably language of the content. The candidates is
 // a map which can be used to assign weights to languages dynamically.
 type Classifier interface {
-	Classify(content []byte, candidates map[string]float64) map[string]float64
+	Classify(content []byte, candidates map[string]float64) (scores map[float64]string)
```

Comment:
Yes it's awkward, I built the map[float64]string to can sort the list of languages by their scores without additionally data structures. I moved the sort logic to the classifier and now the Classifier interface returns a []string of languages.